docs/topics/multiplatform/multiplatform-intro.md

```diff
@@ -62,6 +62,19 @@ For more details on this topic, see:
 * [Add dependencies on Android libraries](multiplatform-android-dependencies.md)
 * [Add dependencies on iOS libraries](multiplatform-ios-dependencies.md)
 
+## Set up integration with iOS
+
+If your multiplatform project targets iOS, you can set up integration of the Kotlin Multiplatform shared module with
+your iOS app.
+
+For that, you generate an iOS framework and then add it as a local or remote dependency to the iOS project:
+
+* **Local integration**: integrate directly using a special script or the CocoaPods dependency manager for setups involving
```

Comment:
I think "integrate" here should be more specific. Maybe rephrase with "integrate something into/with something"?